internal/cli/iam/organizations/organizations.go

```diff
@@ -46,7 +46,7 @@ func AtlasCLIBuilder() *cobra.Command {
 	const use = "organizations"
 	cmd := &cobra.Command{
 		Use:     use,
-		Short:   "Organization operations.",
+		Short:   "Manage your Atlas organizations.",
```

Comment:
Any reference to atlas within IAM will need to be fetched on runtime since the commands are shared between AtlasCLI and MongoCLI. 

Suggestion to check w/ Diancheng for MCLI: "Manage your Ops Manager or Cloud Manager organizations."